WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -612,9 +612,16 @@ private extension NotificationsViewController {
         filterTabBar.items = Filter.allCases
         filterTabBar.addTarget(self, action: #selector(selectedFilterDidChange(_:)), for: .valueChanged)
     }
+}
 
+// MARK: - Jetpack banner UI Initialization
+//
+extension NotificationsViewController {
+
+    /// Called on view load to determine whether the Jetpack banner should be shown on the view
+    /// Also called in the completion block of the JetpackLoginViewController to show the banner once the user connects to a .com account
     func configureJetpackBanner() {
```

Comment:
One thing to check for - if somehow `configureJetpackBanner()` is called multiple times and it passes the guard, would there be a problem calling `addTranslationObserver` if it'd been called before?

Reply:
I guess this maybe deserved a custom `Publisher` instead of a `PassThroughSubject` although that would probably be overkill for this case. 
The double subscription might not be a problem, but maybe we can subscribe on `viewDidLoad` and then conditional call `send(...)` inside `scrollViewDidScroll(...)` based on the usual checks?

Reply:
Both great points guys!  @Gio2018 what you mentioned about subscribe on `viewDidLoad` etc makes sense to me.  I've tested all scenarios and all working as expected.

Reply:
I'd also think that `Combine` is smart enough to hold a dictionary of `Subscribers` of some sort, so it's probably not a huge deal to re-subscribe, cause I'd think there won't be duplicate references (while there can indeed be multiple subscribers).. 🤷‍♂️

Reply:
Just to close the loop from Slack conversations:

We've decided that the risk of over-subscribing to the scroll events is low, and we'll reevaluate the model in the future.